README.md

```diff
@@ -8,45 +8,226 @@
+> Refer to the [HubSpot Developer Terms](https://legal.hubspot.com/developer-terms?_gl=1*fmuxuh*_ga*NDU4NDIxOTA5LjE3MzQ3ODM3Mjg.*_ga_LXTM6CQ0XK*MTczNDgwNjI1OS4yLjAuMTczNDgwNjI1OS42MC4wLjA.&_ga=2.66923502.174841559.1734639641-983296813.1734639641) & [Developer Beta Terms](https://legal.hubspot.com/developerbetaterms?_gl=1*fmuxuh*_ga*NDU4NDIxOTA5LjE3MzQ3ODM3Mjg.*_ga_LXTM6CQ0XK*MTczNDgwNjI1OS4yLjAuMTczNDgwNjI1OS42MC4wLjA.&_ga=2.66923502.174841559.1734639641-983296813.1734639641) for more information.
+>
+> The feedback submissions endpoints are currently read only. Feedback submissions cannot be submitted or edited through the API. But following endpoints are available to view feedback submissions.
+>
+> ```
+> GET https://api.hubapi.com/crm/v3/objects/feedback_submissions
+> GET https://api.hubapi.com/crm/v3/objects/feedback_submissions/{feedbackSubmissionId}
+> POST https://api.hubapi.com/crm/v3/objects/feedback_submissions/batch/read
+> POST https://api.hubapi.com/crm/v3/objects/feedback_submissions/search
+> ```
 
 ## Setup guide
 
-[//]: # (TODO: Add detailed steps to obtain credentials and configure the module.)
+To use the HubSpot Feedback connector, you must have access to the HubSpot API through a HubSpot developer account and a HubSpot App under it. Therefore you need to register for a developer account at HubSpot if you don't have one already.
+
+### Step 1: Create/Login to a HubSpot Developer Account
+
+If you have an account already, go to the [HubSpot developer portal](https://app.hubspot.com/)
+
+If you don't have a HubSpot Developer Account you can sign up to a free account [here](https://developers.hubspot.com/get-started)
+
+### Step 2 (Optional): Create a [Developer Test Account](https://developers.hubspot.com/beta-docs/getting-started/account-types#developer-test-accounts) under your account
+
+Within app developer accounts, you can create developer test accounts to test apps and integrations without affecting any real HubSpot data.
+
+**_These accounts are only for development and testing purposes. In production you should not use Developer Test Accounts._**
+
+1. Go to Test accounts section from the left sidebar.
+   <img src="./docs/resources/test-account.png" style="width: 70%;">
+2. Click on the `Create developer test account` button on the top right corner.
+
+   <img src="./docs/resources/create-test-account.png" style="width: 70%;">
+3. In the pop-up window, provide a name for the test account and click on the `Create` button.
+
+   <img src="./docs/resources/create-account.png" style="width: 70%;">
+4. You will see the newly created test account in the list of test accounts.
+
+   <img src="./docs/resources/test-account-portal.png" style="width: 70%">
```

Comment:
Let's move this to the above step 3, since this is not an actual step where user has to do anything.